src/syntax/syntax.rkt

```diff
@@ -452,47 +550,79 @@
 
 (define-operator (== real real) bool
   ; Override number of arguments
-  [type #hash((* . ((* real) bool)))] [args '(*)]
+  [type #hash((* . (((* real) bool))))] [args '(*)]
   [fl (comparator =)] [bf (comparator bf=)] [cost 65]
   [->c/double (curry format "~a == ~a")]
   [->c/mpfr (curry format "mpfr_set_si(~a, mpfr_cmp(~a, ~a) == 0, MPFR_RNDN)")] ; TODO: cannot handle variary =
   [->tex (infix-joiner " = ")])
 
+(define-operator (complex real real) complex
+  ; Override number of arguments
+  [type #hash((2 . (((real real) complex))))]
+  [fl make-rectangular] [bf bf-make-rectangular] [cost 0]
+  [->c/double (const "/* ERROR: no complex support in C */")] ; TODO
+  [->c/mpfr (const "/* ERROR: no complex support in C */")] ; TODO: cannot handle variary =
```

Comment:
The comment here is incorrect and should be deleted—and similar below.

Reply:
delete TODO at the end